src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
+import { MovieCard } from '../MovieCard';
+import { Movies } from '../../types/Movie';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+export const MoviesList: React.FC<Movies> = ({ allMovie }) => (
+  <>
+    {allMovie.map(movie => (
+      <div className="card" data-cy="Movie" key={movie.imdbId}>
+        <MovieCard {...movie} />
+      </div>
+    ))}
+  </>
```

Comment:
you can make it more semantic. Wrapping into `<ul> ` and `map` `<li>`
```suggestion
  <ul>
    {allMovie.map(movie => (
      <li className="card" data-cy="Movie" key={movie.imdbId}>
        <MovieCard {...movie} />
      </li>
    ))}
  </ul>
```